src/node/stage/mod.rs

```diff
@@ -217,48 +227,57 @@ impl Stage {
         recipient: &SocketAddr,
         msg: Bytes,
     ) -> Result<()> {
-        self.comm.send_message_to_target(recipient, msg).await
+        self.comm.send_message_to_target(recipient, msg).await?;
+        Ok(())
```

Comment:
Is the above change for somehow more clarity and/or preference?

Reply:
No, it would not compile without it. `Comm::send_message_to_target` returns different error type. The `?` operator automatically coverts it to `Error`.